spec/lib/vmdb/config_spec.rb

```diff
@@ -80,4 +80,54 @@
       expect(v).to eq(250.megabytes)
     end
   end
+
+  context "#clone_auth_for_log" do
+    options = {:host => "16.1.2.119",
+               :port => "389",
+               :auth => {:basedn                      => "OU=Users,OU=Developers,OU=Manageiq,DC=manageiq,DC=com",
+                         :bind_dn                     => "CN=Joe Vlcek,OU=Users,OU=Developers,OU=ManageIQ,DC=manageiq,DC=com",
+                         :bind_pwd                    => "top_secret_bind_pwd",
+                         :get_direct_groups           => true,
+                         :group_memberships_max_depth => 2,
+                         :ldaphost                    => ["16.1.2.119"],
+                         :ldapport                    => "389",
+                         :mode                        => "ldap",
+                         :user_suffix                 => "manageiq.com",
+                         :user_type                   => "samaccountname",
+                         :amazon_key                  => nil,
+                         :amazon_secret               => "top_secret_secret",
+                         :ldap_role                   => true,
+                         :amazon_role                 => false,
+                         :httpd_role                  => false,
+                         :user_proxies                => [{:bind_pwd => "user_proxy_bind_pwd", :amazon_secret => "user_proxy_amazon_secret"}],
+                         :follow_referrals            => false,
+                         :sso_enabled                 => false,
+                         :domain_prefix               => "manageiq"
+                        }
+              }
+
+    it "masks_passwors_in_log_auth" do
```

Comment:
Typo `passwors -> password`.  Also, this shouldn't have underscores...just a normal English sentence.